data-hub/src/hub-in-a-box/services/metadata/domain.xml

```diff
@@ -0,0 +1,12 @@
+<!-- Copyright 2016 MarkLogic Corporation.  All Rights Reserved. -->
```

Comment:
Would be cool if (at some point) you had a scriptable test to compare this to what the code provides, to make sure they stay in sync. Might not be easy, though.